<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMDWorkspace</class>
 <widget class="QWidget" name="CreateMDWorkspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>757</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="overviewLayout">
       <item>
        <layout class="QHBoxLayout" name="mementoLoadingLayout">
         <item>
          <widget class="QPushButton" name="btn_add_workspace">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add Workspace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MantidQt::MantidWidgets::WorkspaceSelector" name="workspaceSelector">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_add_nexus_file">
           <property name="text">
            <string>Add Nexus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_add_event_nexus_file">
           <property name="text">
            <string>Add EventNexus File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_remove_workspace">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Remove any workspace once it is highlighted</string>
           </property>
           <property name="text">
            <string>Remove Selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="workspaceEditingLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,1,1,4">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_mandatory">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_latticeCrystal">
                 <item>
                  <widget class="QGroupBox" name="groupBox_lattice">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>2</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>300</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="title">
                    <string>UB Matrix</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QPushButton" name="btn_set_ub_matrix">
                      <property name="text">
                       <string>Set UB Matrix</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QGroupBox" name="gropBox_findUB">
                      <property name="title">
                       <string>Find UB Matrix</string>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_7">
                       <item>
                        <widget class="QPushButton" name="btn_find_ub_matrix">
                         <property name="text">
                          <string>Find UB Matrix</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QGroupBox" name="groupBox_PeakFinding">
                         <property name="title">
                          <string>Peak Finding</string>
                         </property>
                         <layout class="QVBoxLayout" name="verticalLayout_8">
                          <item>
                           <widget class="QRadioButton" name="ckFindSXPeaks">
                            <property name="text">
                             <string>FindSXPeaks</string>
                            </property>
                            <property name="checkable">
                             <bool>true</bool>
                            </property>
                            <property name="checked">
                             <bool>true</bool>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                       <item>
                        <widget class="QGroupBox" name="groupBox_PeakIndexing">
                         <property name="title">
                          <string>Peak Indexing</string>
                         </property>
                         <layout class="QVBoxLayout" name="verticalLayout_9">
                          <item>
                           <widget class="QRadioButton" name="ckIndexPeaks">
                            <property name="text">
                             <string>IndexPeaks</string>
                            </property>
                            <property name="checkable">
                             <bool>true</bool>
                            </property>
                            <property name="checked">
                             <bool>true</bool>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QRadioButton" name="ckIndexSXPeaks">
                            <property name="text">
                             <string>IndexSXPeaks</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                   <zorder>gropBox_findUB</zorder>
                   <zorder>btn_set_ub_matrix</zorder>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QGroupBox" name="logValueGroupBox">
                 <property name="title">
                  <string>Add Log Value</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <item>
                   <widget class="QPushButton" name="btn_add_logs">
                    <property name="text">
                     <string>Add Log Value</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="goniometerGroupBox">
             <property name="title">
              <string>Goniometer Settings</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QPushButton" name="btn_set_goniometer">
                <property name="text">
                 <string>Set Goniometer</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="creationGroupBox">
             <property name="title">
              <string>Create MD Workspace</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QCheckBox" name="ck_keep">
                <property name="text">
                 <string>Keep generated workspace in memory</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="mergeLayout">
                <item>
                 <widget class="QCheckBox" name="ck_merge">
                  <property name="text">
                   <string>Merge All MD Workspaces</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="lbl_merged_name">
                  <property name="text">
                   <string>name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txt_merged_workspace_name">
                  <property name="toolTip">
                   <string>Name for the merged workspace</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="btn_create">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="toolTip">
                 <string/>
                </property>
                <property name="text">
                 <string>Create MD Workspace(s)</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="spacerLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlButtonsLayout"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::WorkspaceSelector</class>
   <extends>QComboBox</extends>
   <header>MantidQtMantidWidgets/WorkspaceSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
